utils/api/configurations.js

```diff
@@ -75,6 +75,61 @@ const normalizeDate = (str) => {
   return `${date.toDateString().substring(3)} at ${date.toLocaleTimeString()}`
 }
 
+export const timeSince = function(date) {
+  if (typeof date !== 'object') {
+    date = new Date(date);
```

Comment:
running lint on this file should have removed all the semi colons.

Reply:
![image](https://user-images.githubusercontent.com/73361970/209398419-f787c3e1-29ea-4542-a13c-59ea5aa4cac2.png)

when i run yarn lint i get no warnings or errors. and when i run `npx eslint utils/api/configurations.js` i get nothing as well. maybe we need to update a rule?

Reply:
hmmm. surprising. perhaps. ok I'll leave it for now and check the rules in a diff pr. 🙂